### ISSUE TITLE
TypeError: avro.Type.forSchema(...) is not a function

### ISSUE THREAD

**Issue by 1jerry** [mtth/avsc#248]
Error on trying example code.  Second line of code, after ```const arvo = require('avsc')```

avsc 5.4.12

**Comment by mtth** [comment]
Is this on `node`? Which version are you using? Can you share a snippet reproducing the error?

**Comment by 1jerry** [comment]
Yes, I was running the script using node.js.  I agree that this doesn't make sense so I did some more digging and found the problem, in case anyone else runs into this.
Here is the code that will create the error:
```
const avro = require('avsc')

const type = avro.Type.forSchema({
    "type": "record",
    "fields": [
        {
            "name": "Caller",
            "type": "string",
        },
    ]
})

(async () => {
    console.log('worked')
})()
```
Can you see anything obvious?  I didn't either.  So I added a console.log before the anonymous function at the end, and the error changed to "console.log(...) is not a function".  Aha!  The original error is not saying _forSchema_ is not a function, it is saying _forSchema()_ is not a function.

### TL;DR
Make sure there is a semi-colon (;) _before_ any statement that begins with a parenthesis "(".  It had nothing to do with avsc.  You can also surround the whole three lines at the end with { and } to make it an anonymous function inside an anonymous name space if you don't want to see a semi-colon.
